eli5/ipython_image.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from typing import Any
+
+from PIL import Image # type: ignore
+
+from .base import Explanation
+from .formatters import format_as_image
+
+
+def display_prediction_image(expl, **format_kwargs):
```

Comment:
This function does nothing except for calling ``format_as_image``, do we need it?

Reply:
The original idea was to use separate files because the image display method has extra dependencies, and it is not related to the HTML display method. Though for this PR I think it's fine keeping everything in one file.

https://github.com/TeamHG-Memex/eli5/pull/315/commits/998d135d7d050fd23c0a10e065f840555d1d6cee